build.sbt

```diff
@@ -8,14 +8,14 @@ import bijection._
 val finagleVersion210 = "6.35.0"
 val finagleVersion = "6.41.0"
```

Comment:
Latest finagle is also 19.9.0 now and built for scala 2.13. Should we use that?
https://mvnrepository.com/artifact/com.twitter/finagle-core

Reply:
19.10.0 should be on its way. I can put everything on that if desirable and we can hold off for a few days.

Reply:
indeed, 19.10.0 has been published, looks like mvnrepository might have a stale cache https://search.maven.org/artifact/com.twitter/finagle-core_2.13/19.10.0/jar

Reply:
I consolidated on 19.10.0